terraform/secops/secops-iam-policies.tf

```diff
@@ -0,0 +1,360 @@
+                "ssm:StopAutomationExecution",
+                "elasticbeanstalk:*",
+                "ssm:DescribeInstanceInformation",
+                "ssm:ListTagsForResource",
+                "ssm:CreateAssociationBatch",
+                "ssm:ModifyDocumentPermission",
+                "ssm:DescribeDocumentParameters",
+                "ecr:*",
+                "ssm:RegisterDefaultPatchBaseline",
+                "ssm:UpdateInstanceAssociationStatus",
+                "acm:*",
+                "ssm:DeregisterTargetFromMaintenanceWindow",
+                "ssm:ListDocumentVersions",
+                "ssm:UpdateDocumentDefaultVersion",
+                "logs:*",
+                "ssm:DescribeInstancePatches",
+                "autoscaling:*",
+                "ssm:UpdateMaintenanceWindowTarget",
+                "ssm:GetMaintenanceWindowExecutionTaskInvocation",
+                "ssm:DeleteActivation",
+                "cloudfront:*",
+                "ssm:StartAssociationsOnce",
+                "ssm:UpdateInstanceInformation",
+                "ssm:AddTagsToResource",
+                "opsworks:*",
+                "config:*",
+                "ssm:PutComplianceItems",
+                "events:*",
+                "inspector:*",
+                "directconnect:*",
+                "cloudformation:*",
+                "ssm:RegisterTargetWithMaintenanceWindow",
+                "iam:*",
+                "waf:*",
+                "servicecatalog:*",
+                "ssm:ListCommandInvocations",
+                "lightsail:*",
+                "shield:*",
+                "ssm:DescribeMaintenanceWindowTasks",
+                "ssm:GetPatchBaselineForPatchGroup",
+                "ssm:DeletePatchBaseline",
+                "ssm:DescribeMaintenanceWindowExecutions",
+                "ssm:GetManifest",
+                "ds:*",
+                "batch:*",
+                "ssm:DescribeInstancePatchStates",
+                "artifact:*",
+                "ssm:DeleteResourceDataSync",
+                "ssm:DeregisterTaskFromMaintenanceWindow",
+                "ssm:DescribeInstanceAssociationsStatus",
+                "ssm:DescribeInstanceProperties",
+                "ssm:GetDeployablePatchSnapshotForInstance",
+                "ssm:DescribeMaintenanceWindowTargets",
+                "cloudhsm:*",
+                "ssm:DeleteMaintenanceWindow",
+                "ssm:DescribeEffectiveInstanceAssociations",
+                "ssm:GetInventory",
+                "ssm:DescribeActivations",
+                "ssm:GetCommandInvocation",
+                "ssm:DeleteAssociation",
+                "ssm:DeregisterManagedInstance",
+                "trustedadvisor:*",
+                "ssm:PutInventory",
+                "sso:*",
+                "route53:*",
+                "ssm:ListAssociations",
+                "ssm:DeleteDocument",
+                "ssm:DescribeAvailablePatches"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+EOF
```

Comment:
Not a blocker, but I recommend saving the policies in the repository as JSON files, and including them using something like

```hcl
policy = "${file("${path.module}/admin_policy.json")}"
```

Makes the JSON syntax-highlighted, and keeps the Terraform shorter / more readable.